app/client/src/components/CustomCard/CardContent.tsx

```diff
@@ -1,10 +1,10 @@
-import { Tooltip, Link, Typography, Button } from "@material-ui/core";
-import React from "react";
-import { formatCount } from "../../utils";
-import { CardProps } from "./model";
-import { useStyles } from "./styles";
-import InfoIcon from "@material-ui/icons/Info";
-import clsx from "clsx";
+import { Tooltip, Link, Typography, Button } from '@material-ui/core';
```

Comment:
Unnecessary changes, will make things difficult to debug later. Just send the changes for css that you have